modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,133 @@ endif::[]
 :meta-product: API Manager, Studio, Mule
 :meta-applies-to:
 
-Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any organization.
+Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any API Platform organization (including the Federated ones). It is the recommended initial solution pending future corporate investments in Enterprise specific OAuth 2.0 offerings like
+PingFederate, OpenAM or OAuth providers supporting Open ID Connect.
 
-In addition to Mule OAuth 2.0 Provider, MuleSoft offers out-of-the-box integration with PingFederate, OpenAM and Open ID Connect OAuth 2.0 Providers.
+The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] is designed to work with the Mule OAuth 2.0 provider.
+
+== Getting the Mule OAuth 2.0 provider
+To run the Mule OAuth 2.0 Provider you have to deploy the provider to a runtime with API Gateway capabilities (either Mule runtime engine v3.x, Mule runtime engine v4.2.0 or later, or API Gateway v2.x.x) with the corresponding organization credentials correctly specified.
```

Comment:
`, or API Gateway v2.x.x ` you name this here but there is no later section describing how to setup like with 3.x and 4.x, if it is the same as 3.x you can add the entry saying it is the same. (or remove it from here ;P )

Reply:
Done.